sources/packages/web/src/components/common/StudentAppealForm.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
We talked briefly about this component receiving an array of _formNames_ and doing the "for" to render all of them internally. You did not pursue this approach for some specific reason?

Reply:
May be there is a gap in communication. I was in a understanding that we are using a component to represent an appeal form and iterate this component with the list of formnames we get. We can sync up to get it cleared out.

Reply:
We talked about two possible ways:
1. Do the v-for where it is right now
2. Try to do it inside the component.
The idea was to prepare the component to be reused for the Ministry, right?

Reply:
As per the conversation, let's try to have the code centralized as much as possible when the upcoming Ministry view came. The effort so far was already pretty good.